src/vnsw/opencontrail-vrouter-netns/opencontrail_vrouter_netns/vrouter_hyperv.py

```diff
@@ -218,21 +279,19 @@ def _get_all_mgmt_ips(self):
             mgmt_ips = []
         return mgmt_ips
 
-
     def _get_mgmt_ips_of(self, vmname_with_wildcard):
-        ips = powershell(["Get-VMNetworkAdapter",
-                          "-VMName", vmname_with_wildcard, "|",
-                          "Where", "SwitchName", "-eq", self.mgmt_vswitch_name,
-                          "|", "Select", "-ExpandProperty", "IPAddresses"])
+        ips = call_powershell(["Get-VMNetworkAdapter",
```

Comment:
I don't think unit tests for glue code are worth the effort tbh... if anything, I would do some integration tests against Hyper-V, but they aren't worth it either I think.